### PR TITLE
Feature: Make the text a bit lighter

### PR DESCRIPTION
**Pull request by ArudoPE**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behaviour
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
  - `Fix` - bug fixes

Complete the following checkbox ONLY IF it is applicable to your PR. You can complete it later if it is not currently applicable:
-   [x] I have verified all tests and linters pass against my changes, and/or I have included automated tests where applicable

<hr>

**1. Because:**
* The text on the site was a bit hard to read because the color is close to gray.

**2. This PR:**
-  Make the p elements color a bit lighter (from #b3b3b3 to #cac7c7)

Before and after

### CODE REVIEW

File: app/assets/stylesheets/dark-mode.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 :root {
   --dark-bg-color: #121212;
   --dark-bg-accent: #1a1a1a;
-  --dark-font-color: #b3b3b3;
+  --dark-font-color: #cac7c7;
```
Comment (KevinMulhern) [inline app/assets/stylesheets/dark-mode.css:4]: Hey @ArudoPE thanks for doing this. However, I don't think that color fits. It looks off when paired with headings.  

Playing around with it, white with a slightly reduced opacity seems to work better:
```suggestion
  --dark-font-color: rgba(255, 255, 255, 0.87);
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/dark-mode.css (hunk 1 above)
Comment (ArudoPE) [reply app/assets/stylesheets/dark-mode.css:4]: Yes, actually, that would be even better. I tried to not get too close to white, but that's a nice suggestion.

**Review by KevinMulhern — COMMENTED** [review]
Thanks @ArudoPE, this looks good. Before we merge, can you squash the second commit into the first one please?

**Comment by ArudoPE** [comment]
> Thanks @ArudoPE, this looks good. Before we merge, can you squash the second commit into the first one please?

I'm very sorry but since I'm kind of new to GitHub I don't know how to do that

**Comment by KevinMulhern** [comment]
No problem, this [video](https://www.youtube.com/watch?v=2E23I9PzplM) or this [thread](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) should help

**Comment by ArudoPE** [comment]
> No problem, this [video](https://www.youtube.com/watch?v=2E23I9PzplM) or this [thread](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) should help

Is it ok now? I'm not sure if I did what I needed to do.

**Comment by KevinMulhern** [comment]
Hey @ArudoPE, no unfortunately not, there are three commits now instead of one. An empty merge commit has been added. We only want to have one commit on this since it's a small one-line change and the extra commits are from fixing review feedback. 

I just noticed you're doing a patch through the GitHub UI. You will need to squash the commits with a forked version of the repo through the command line. The video posted earlier is a good guide for what to do.

Alternatively, you could open a new PR with the changes in one commit.

**Comment by ArudoPE** [comment]
> Hey @ArudoPE, no unfortunately not, there are three commits now instead of one. An empty merge commit has been added. We only want to have one commit on this since it's a small one-line change and the extra commits are from fixing review feedback.
> 
> I just noticed you're doing a patch through the GitHub UI. You will need to squash the commits with a forked version of the repo through the command line. The video posted earlier is a good guide for what to do.
> 
> Alternatively, you could open a new PR with the changes in one commit.

Yes, I'm gonna open a new PR.

**Pull request closed without merging**